File: baekjoon/basic2/1436/1436.py
# 백준 1436 : 영화감독 숌 (https://www.acmicpc.net/problem/1436)
import sys
    
n = int(sys.stdin.readline())
i = 666
cnt = 0
while True:
    if '666' in str(i):
        cnt += 1
    if cnt == n:
        break
    i += 1
sys.stdout.write(str(i)+"\n")

File: baekjoon/basic2/1436/1436.cpp
// 백준 1436 : 영화감독 숌 (https://www.acmicpc.net/problem/1436)
#include <iostream>
#include <string>
using namespace std;

int main(){
    ios_base::sync_with_stdio(false);
    cin.tie(NULL); cout.tie(NULL);
    int n;
    cin >> n;
    int i = 666;
    int cnt = 0;
    for(;; i++){
        if(to_string(i).find("666") != string::npos) cnt++;
        if(cnt == n) break;
    }
    cout << i << "\n";

    return 0;
}